common/txmgr/broadcaster.go

```diff
@@ -694,37 +709,33 @@ func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) tryA
 			attempt.TxFee, txError.Error(), eb.feeConfig.FeePriceDefault())
 	}
 
-	replacementAttempt, bumpedFee, bumpedFeeLimit, retryable, err := eb.NewBumpTxAttempt(ctx, etx, attempt, nil, lgr)
+	bumpedAttempt, bumpedFee, bumpedFeeLimit, _, err := eb.NewBumpTxAttempt(ctx, etx, attempt, nil, lgr)
 	if err != nil {
-		return fmt.Errorf("tryAgainBumpFee failed: %w", err), retryable
+		return bumpedAttempt, fmt.Errorf("replaceAttemptWithBumpedGas failed: %w", err)
```

Comment:
Extending the existing error message should be used if we want to add more information to it that is valuable to the reader. We don't have to add the function's name to the error message as this can be retrieved by the stack trace. We mistakenly do this across the codebase, so let's change it here:
```suggestion
		return bumpedAttempt, err
```